scripts/glean.py

```diff
@@ -49,11 +49,7 @@ def __init__(self, identifier: str, definition: dict, *, ping_names: List[str] =
         self._set_definition(definition)
         self._set_description(self.definition)
 
-        defn_pings = set(
-            [p for d in definition[self.HISTORY_KEY] for p in d.get("send_in_pings", ["metrics"])]
-        )
```

Comment:
I am pretty sure this type of logic is unnecessary-- by the time probe-scraper outputs the files for us to consume, the defaults should already be set:

https://github.com/mozilla/glean_parser/blob/f9b676646f82d93ca2698540646a6d0bd29ab3ad/glean_parser/parser.py#L320